src/models/simulation.ts

```diff
@@ -334,19 +342,28 @@ export class SimulationModel {
           // rendering code is calculating colors for mesh faces.
           const isNonBurnable = this.config.fillTerrainEdges &&
             x <= 1 || x >= this.gridWidth - 2 || y <= 1 || y >= this.gridHeight - 2;
+          // store zone index for tracking burn percentage
+          const zi = zoneIndex ? zoneIndex[index] : 0;
           const cellOptions: CellOptions = {
             x, y,
-            zone: this.zones[zoneIndex ? zoneIndex[index] : 0],
+            zone: this.zones[zi],
+            zoneIdx: zi,
             isRiver,
             isUnburntIsland: unburntIsland && unburntIsland[index] > 0 || isNonBurnable,
             baseElevation: isEdge ? 0 : elevation && elevation[index]
           };
+          if (!this.totalCellCountByZone[zi]) this.totalCellCountByZone[zi] = 1;
+          else {
+            this.totalCellCountByZone[zi]++;
+          }
           this.cells.push(new Cell(cellOptions));
         }
       }
       this.updateCellsElevationFlag();
       this.updateCellsStateFlag();
       this.updateTownMarkers();
+      // dimensions in feet, convert sqft to acres
+      this.simulationAreaAcres = this.gridWidth * this.gridHeight * this.cellSize / 43560;
```

Comment:
I thought I added a comment about that, but maybe it got lost somewhere. I think this equation is incorrect. It's missing one ` * this.cellSize` to have correct units. Or, it can be simplified by using model dimensions in feet:
`this.config.modelWidth * this.config.modelHeight / 43560 `.

Reply:
Thanks! I did miss this one, have now pushed the change.